src/main/resources/org/gradle/api/internal/plugins/windowsStartScript.txt

```diff
@@ -17,6 +17,8 @@ if "%DIRNAME%" == "" set DIRNAME=.\
 set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%${appHomeRelativePath}
 
+set "DEFAULT_JVM_OPTS=%DEFAULT_JVM_OPTS% -Dpa.scheduler.home="%APP_HOME%""
```

Comment:
In cmd.exe you can't properly escape double quotes. It's the best solution I found, but I haven't tested in path containing spaces (which I should I agree)